readme.md

```diff
@@ -357,6 +357,7 @@
 - [Construct 2](https://github.com/WebCreationClub/awesome-construct#readme) - Game engine.
 - [Gideros](https://github.com/stetso/awesome-gideros#readme) - Game engine.
 - [Minecraft](https://github.com/bs-community/awesome-minecraft#readme) - Sandbox video game.
+- [Haxe Game Development](https://github.com/Dvergar/awesome-haxe-gamedev#readme) - Game development for Haxe, the cross-platform toolkit.
```

Comment:
It's already clear from the title that it's game development, so I would rather write what Haxe is, using a few more words.

Reply:
This still needs to be addressed, then this MR looks good to me.

Reply:
Already addressed https://github.com/sindresorhus/awesome/pull/1573/commits/dfb532a77f76407236cf02f1402fe715865a5a7c

Reply:
What about putting it under the "Game Development" item further up in the list?